src/BinaryParsers/ElfBinary/Dwarf/DwarfSymbolProvider.cs

```diff
@@ -58,6 +94,80 @@
             return programs;
         }
 
+        /// <summary>
+        /// Parses all command line info.
+        /// </summary>
+        /// <param name="compilationUnits">the compilation units.</param>
+        internal static List<DwarfCompileCommandLineInfo> ParseAllCommandLineInfos(List<DwarfCompilationUnit> compilationUnits)
+        {
+            List<DwarfCompileCommandLineInfo> returnValue = new List<DwarfCompileCommandLineInfo>();
+
+            foreach (DwarfCompilationUnit compilationUnit in compilationUnits)
+            {
+                foreach (DwarfSymbol symbol in compilationUnit.Symbols)
+                {
+                    DwarfCompileCommandLineInfo info = new DwarfCompileCommandLineInfo();
+                    info.FullName = symbol.Attributes.ContainsKey(DwarfAttribute.Name)
```

Comment:
>ContainsKey [](http://example.com/codeflow?start=54&length=11)

replace for trygetvalue, that will prevent checking if contains and getting the value.

apply this for the other places we can in this file

Reply:
got it will find and change all.

Reply:
I have changed from using ContainsKey to TryGetValue